src/main/java/com/google/sps/data/User.java

```diff
@@ -22,11 +29,35 @@ public boolean equals(User user) {
   }
 
   public void addProductArea(String toAdd) {
-      productArea.add(toAdd);
+      specialties.add(toAdd);
   }
 
-  public Collection<String> getProductAreas() {
-      return productArea;
+  public Collection<String> getProductArea() {
```

Comment:
I would rename to getSpecialties()

Reply:
I also did this in my PR so if I merge before you, it will hopefully update

Reply:
I will later, for now it has to be that due to other code relying on get product area

Reply:
Oh nvm  i can do that